pkg/storage/sqlite/sqlite.go

```diff
@@ -60,29 +63,89 @@ func (s SQLite) CreateIndexer(ctx context.Context, indexer model.Indexers) (int6
 
 // DeleteIndexer deletes a stored indexer given the indexer ID
 func (s SQLite) DeleteIndexer(ctx context.Context, id int64) error {
-	stmt := table.Indexers.DELETE().WHERE(table.Indexers.ID.EQ(sqlite.Int64(id))).RETURNING(table.Indexers.AllColumns)
+	stmt := table.Indexer.DELETE().WHERE(table.Indexer.ID.EQ(sqlite.Int64(id))).RETURNING(table.Indexer.AllColumns)
```

Comment:
Phew I wasn't a plural fan

Reply:
I swapped your new schema out of plural as well fyi